server/tests/io/feed_parsers/belga_tass_newsml_1_2_test.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013 - 2018 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+
+import os
+from lxml import etree
+from superdesk.tests import TestCase
+from belga.io.feed_parsers.belga_tass_newsml_1_2 import BelgaTASSNewsMLOneFeedParser
+
+
+class BelgaTASSNewsMLOneTestCase(TestCase):
+    filename = 'tass_belga.xml'
+
+    def setUp(self):
+        dirname = os.path.dirname(os.path.realpath(__file__))
+        fixture = os.path.normpath(os.path.join(dirname, '../fixtures', self.filename))
+        provider = {'name': 'test'}
+        with open(fixture, 'rb') as f:
+            parser = BelgaTASSNewsMLOneFeedParser()
+            self.xml_root = etree.parse(f).getroot()
+            self.item = parser.parse(self.xml_root, provider)
+
+    def test_can_parse(self):
+        self.assertTrue(BelgaTASSNewsMLOneFeedParser().can_parse(self.xml_root))
+
+    def test_content(self):
+        item = self.item[0]
+        self.assertEqual(item["role"], "Main")
+        self.assertEqual(item["slug_line"], "URGENT")
+        self.assertEqual(item["dateline"], {'text': 'January 21'})
+        self.assertEqual(item["genre"], [])
+        self.assertEqual(item["news_item_type"], "News")
+        self.assertEqual(item["products"], ['FILE_MROUTER'])
+        self.assertEqual(item["news_component_essential"], "no")
+        self.assertEqual(item["guid"], "03AE4325838900396A95")
+        self.assertEqual(item["date_line"], "January 21")
+        self.assertEqual(item["language"], "en")
+        self.assertEqual(item["news_component_equivalentslist"], "no")
+        self.assertEqual(item["headline"], "Kremlin has 'negative reaction' to upcoming EU sanctions")
+        self.assertEqual(item["link_type"], "normal")
+        self.assertEqual(item["location"], {'how_present_el': 'Origin', 'city': 'MOSCOW'})
+        self.assertEqual(item["subject"], [])
+        self.assertEqual(item["item_keywords"], ['itartassrubric_URGENT', 'URGENT'])
+        self.assertEqual(item["mime_type"], "text/vnd.IPTC.NITF")
+        self.assertEqual(item["type"], "text")
+        self.assertEqual(item["urgency"], "")
+        self.assertEqual(item["pubstatus"], "Usable")
+        self.assertEqual(item["provider_id"], "\nwww.itar-tass.com\n")
+        self.assertEqual(item["versioncreated"].isoformat(), "2019-01-21T10:27:08")
+        self.assertEqual(item["firstcreated"].isoformat(), "2019-01-21T10:27:08")
```

Comment:
it's missing timezone info, we should convert it to utc so it will work on client, I guess they are using local time